.github/workflows/interchaintest.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Set up Go 1.20
```

Comment:
same as my last comment, maybe we quit referencing a specific Go version in the names that way we don't need to keep making changes every time we update the Go version.